iarrc/launch/nn_planner_circuit.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+    <include file="$(find rr_common)/launch/nn_planner.launch">
+        <arg name="image_topic" value="camera/image_color_rect_flipped"/>
```

Comment:
are we using the old /image_rect topic? if not we might want to remap image_color_rect_flipped into that. The latter is unnecessarily verbose imo

Reply:
image_color_rect_flipped just stems from the ROS conventions on building names by appending the operations done.
We'll be able to get rid of the _flipped part anyway as we're moving the camera to the other side of the plate it's mounted to, which will put it back to being upright.
Whether we keep the _color in there is up to us. It distinguishes it from the monochromatic topic that [image_proc/debayer](http://wiki.ros.org/image_proc) publishes.